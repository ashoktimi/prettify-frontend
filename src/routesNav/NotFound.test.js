import React from "react";
import { render } from "@testing-library/react";
import NotFound from './NotFound';
import { MemoryRouter } from "react-router";

it("matches snapshot", function () {
  const { asFragment } = render(
      <MemoryRouter>
        <NotFound />
      </MemoryRouter>,
  );
  expect(asFragment()).toMatchSnapshot();
});
