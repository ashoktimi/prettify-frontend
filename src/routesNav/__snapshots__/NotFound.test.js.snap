// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div>
    <h1>
      404 Not Found
    </h1>
    <p>
      The page you are looking for could not be found.
    </p>
  </div>
</DocumentFragment>
`;
